src/Controllers/ClassroomsController.cs

```diff
@@ -91,5 +91,34 @@ public async Task<JsonResult> GetPhysicsSubmissionOfTheStudent(string studentId,
                 });
             }
         }
+
+        public async Task<JsonResult> PostPhysicsSubmissionOfTheStudent(string SubmitStatus, string postJsonPhy, string submissionID)
+        {
```

Comment:
Could be pushed a "Submission submission" object from front-end. But it's okay for now.